### PR TITLE
ISSUE::> Add missing methods for new fedimint-http methods DONE...

### PR DESCRIPTION
**Pull request by PsychoPunkSage**
> Added new Functions:

- getActiveFederationId()
- setActiveFederationId()
- Backup()
- Join()
- FederationIds()

> Updated all the functions of `Mint`, `Wallet`, `ln` to make them use `federationIds`

### CODE REVIEW

File: pkg/fedimint/client.go [hunk 1]
```diff
@@ -133,12 +153,45 @@ func (fc *FedimintClient) Config() (*types.FedimintResponse, error) {
 	return &configResp, nil
 }
 
+func (fc *FedimintClient) Join(inviteCode string, setDefault bool) (types.FederationIdsResponse, error) {
+	var response types.FederationIdsResponse
+	responseBody, err := fc.post("/admin/backup", map[string]interface{}{
```
Comment (Kodylow) [inline pkg/fedimint/client.go]: this is wrong, needs to call join not backup

**Review by Kodylow — CHANGES_REQUESTED** [review]

File: pkg/fedimint/client.go (hunk 1 above)
Comment (PsychoPunkSage) [reply pkg/fedimint/client.go]: Oh,
Sorry, my bad. Will change it

**Comment by PsychoPunkSage** [comment]
I made required changes...
Are there any more issues?

**Comment by Kodylow** [comment]
it looks okay, I'll merge, would be good to get some test scripts to cover this though

**Comment by Kodylow** [comment]
can you rebase before merge though

**Comment by PsychoPunkSage** [comment]
Okay,
Like I have to do something like this `git rebase master` > `git push origin master`? right?

**Comment by PsychoPunkSage** [comment]
I have done it.

**Review by Kodylow — APPROVED** [review]

**Pull request merged**